From 9a0103f97265f405b70891f46d67e2944ac42f11 Mon Sep 17 00:00:00 2001
From: Manuel Fombuena <fombuena@outlook.com>
Date: Sun, 28 Jan 2024 16:42:34 +0000
Subject: soc: qcom: socinfo: Add IDs for IPQ8174 family

Ported from https://lore.kernel.org/all/20230901181041.1538999-2-robimarko@gmail.com/

IPQ8174 (Oak) family is part of the IPQ8074 family, but the ID-s for it
are missing so lets add them.

Signed-off-by: Manuel Fombuena <mfombuena@innovara.co.uk>
---
 drivers/soc/qcom/socinfo.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/soc/qcom/socinfo.c
+++ b/drivers/soc/qcom/socinfo.c
@@ -250,6 +250,9 @@ static const struct soc_id soc_id[] = {
 	{ 394, "SM6125" },
 	{ 395, "IPQ8070A" },
 	{ 396, "IPQ8071A" },
+	{ 397, "IPQ8172" },
+	{ 398, "IPQ8173" },
+	{ 399, "IPQ8174" },
 	{ 402, "IPQ6018" },
 	{ 403, "IPQ6028" },
 	{ 421, "IPQ6000" },
